apache-rat-core/src/main/java/org/apache/rat/Defaults.java

```diff
@@ -143,10 +114,10 @@ public static void init() {
     /**
      * Builder constructs instances.
      * @param log The log to write messages to.
-     * @param uris The set of URIs to read.
+     * @param urls The set of URLs to read.
      */
-    private Defaults(final Log log, final Set<URI> uris) {
-        this.setFactory = Defaults.readConfigFiles(log, uris);
+    private Defaults(final Log log, final Set<URI> urls) {
```

Comment:
I thought the other merge part 1 already fixed this and changed it into uris instead of urls? Is that a sideeffect of the forced push you did on the branch8es)?